demo/admin/src/products/future/ProductForm.cometGen.ts

```diff
@@ -35,6 +35,15 @@ export const ProductForm: FormConfig<GQLProduct> = {
             collapsible: true,
             initiallyExpanded: false,
             fields: [
+                {
+                    type: "asyncSelect",
+                    name: "manufacturer",
+                    rootQuery: "manufacturers",
+                    filterField: {
+                        name: "type",
+                        filterMapping: `addressAsEmbeddable_country: { equal: {value} }`,
```

Comment:
Is this necessary? Won't this be equal all the time?

Reply:
that's right. the hard part to guess is `addressAsEmbeddable_country`. I thought it would be more flexible, e.g. supporting deeper nested filters, if filterMapping contained to whole string with a value-placeholder. but I can change it to filterName and only require to set `addressAsEmbeddable_country`. this will probably still cover all cases.

Reply:
it probably has to support another case. I'm pretty sure I'll also need to support adding the filterValue as root-prop, e.g. membersByClub(club), not only in filter.

Reply:
> it probably has to support another case. I'm pretty sure I'll also need to support adding the filterValue as root-prop, e.g. membersByClub(club), not only in filter.

This is out of scope of this PR. Please remove `gqlVarType` here, and create a separate PR where we discuss this (with a concrete example).